pandas/_config/config.py

```diff
@@ -818,7 +823,7 @@ def inner(x):
     return inner
 
 
-def is_nonnegative_int(value):
+def is_nonnegative_int(value: Optional[int]) -> None:
```

Comment:
strange that this returns None. I would expect a function named is_* to return True/False.

Reply:
I would be OK to rename this `check_nonnegative_int` in another PR